app/common/update.py

```diff
@@ -253,3 +258,65 @@ def merge_local_db():
 
         logger.info(str(records_inserted) + " records were inserted into local db.")
         logger.info(str(records_updated) + " records in local db were updated.")
+        
+
+def download_dat_files():
+    config = load_user_config()
+    valid_directory = False
+    
+    if not config["config"]["installdirectory"]:
+    
+        #First, let's check the default location and see if it's installed there

```

Comment:
Can you add spaces after `#`? More for consistency across the rest of the project. I'll just add it here, but if you could update it in all instances.